dbt_project.yml

```diff
@@ -25,6 +25,7 @@ vars:
     bundle: "{{ source('quickbooks','bundle') }}"
     credit_memo_line: "{{ source('quickbooks','credit_memo_line') }}"
     credit_memo: "{{ source('quickbooks','credit_memo') }}"
+    credit_card_payment_txn: "{{ source('quickbooks','credit_card_payment_txn') }}"
```

Comment:
I noticed there isn't a table locally in `quickbooks_customer` for `credit_card_payment_txn`. Where do we find the  table name value in that case? Is it in the Fivetran connector?

Reply:
This table is not included in all QuickBooks schemas. This is only included if customers use credit_card_payments. This is true for all other QuickBooks tables. 

While we do not have this table locally, we were able to develop the table through the help of the community within [Issue #47](https://github.com/fivetran/dbt_quickbooks/issues/47)

Reply:
Nice, looks fairly thorough.